optimum/commands/env.py

```diff
@@ -0,0 +1,98 @@
+# Copyright 2024 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import platform
+import subprocess
+from typing import Dict, List, Optional
+
+import huggingface_hub
+from pkg_resources import get_distribution
+from transformers import __version__ as transformers_version
+from transformers.utils import is_torch_available
+
+from ..neuron.utils import is_neuron_available, is_neuronx_available
+from ..neuron.version import __sdk_version__ as neuron_sdk_version
+from ..neuron.version import __version__ as optimum_neuron_version
+from ..version import __version__ as optimum_version
+from . import BaseOptimumCLICommand, CommandInfo
+
+
+class EnvironmentCommand(BaseOptimumCLICommand):
+    COMMAND = CommandInfo(name="env", help="Get information about the environment used.")
+
+    @staticmethod
+    def format_dict(d):
+        return "\n".join([f"- {prop}: {val}" for prop, val in d.items()]) + "\n"
+
+    @staticmethod
+    def get_pip_pkgs_version(pkg_list: Optional[List], info: Dict):
+        if pkg_list is not None:
+            for pkg in pkg_list:
+                try:
+                    num_version = get_distribution(pkg).version
+                except Exception:
+                    num_version = "NA"
+                info[f"`{pkg}` version"] = num_version
+
+    @staticmethod
+    def print_apt_pkgs():
+        info = subprocess.run("apt list --installed | grep aws-neuron", capture_output=True, shell=True)
```

Comment:
nit: Would not use `shell=True` unless absolutely necessary.

Reply:
My previous concern was without `shell=True`, it captures only staging outputs, something like:
```
Neuron Driver:

Listing...
```
instead of the neuron driver information that I want to show... There is `subprocess.getoutput` but under the hood, it still can only execute the cmd in a shell as well. 

I just put a workaround to avoid shell=True.